SalesforceSDK/Salesforce.Universal.Template/Salesforce.Universal.Template.Shared/Pages/MainPage.xaml.cs

```diff
@@ -53,11 +57,41 @@ namespace $safeprojectname$.Pages
     /// </summary>
     public sealed partial class MainPage : NativeMainPage
     {
+        private const string ApiVersion = "v31.0";
```

Comment:
Actually there is a ApiVersionStrings.VersionNumber that does this. I'll change it to use that